### ISSUE TITLE
Add position feature

### ISSUE THREAD

**Issue by fharper** [mosen/mysides#26]
Right now if I want to change the order of some favorites I have to add and remove until I get the right order. It would be nice to be able to automate that process with a command like `mysides position <position> <name>` as  `mysides position 2 Desktop` to move my Desktop favorites from actual position to second from the top.

**Comment by mosen** [comment]
Hiya, that sounds pretty helpful. I'm sorry I have not been working on this for a long time, but i will see if i can get some time to add this feature. I'm surprised if it still works in Monterey

**Comment by opus-nbonenberger** [comment]
@mosen I would like to see a position or move feature, too. You could do it like here: https://github.com/Ajordat/finder-sidebar-editor

I'd rather use your tool instead of Finder-Sidebar-Editor because this installs a huge load of Python modules.

**Comment by mosen** [comment]
Yeah can do. Haven't looked at the code for a while but it might be worth a shot

**Comment by opus-nbonenberger** [comment]
@mosen Great! Can you mention here when you have implemented the feature? Thanks!
